### PR TITLE
L10n fr

### PR DESCRIPTION
**Pull request by tdd**
Hey again,

As mentioned in #38, this is the second step of our recent work on globalizing and localizing `promise-it-wont-hurt`. This includes the commits referenced in #38, which will "disappear" when/if you merge that one first, to leave only FR-related commits for actual localization.

I remain, naturally, available for any discussion on this.

Looking forward to our getting that in,

Best,

### CODE REVIEW

File: exercises/more_functional/problem.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Let's tweak/refactor the previous lesson to be more declarative!
```
Comment (TimothyGu) [inline exercises/more_functional/problem.md]: I'm planning on dropping this exercise. See #48. What do you think?

Plus this line should probably be "more functional!"

File: promiseitwonthurt.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
+
+const workshopper = require('workshopper')
+    , path        = require('path')
+
+function fpath (f) {
+  return path.join(__dirname, f)
```
Comment (TimothyGu) [inline promiseitwonthurt.js]: Nit:

```js
fpath = path.join.bind(null, __dirname)
```

File: promiseitwonthurt.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+}
+
+workshopper({
+    name        : 'promise-it-wont-hurt'
+  , appDir      : __dirname
+  , languages   : ['en', 'fr']
```
Comment (TimothyGu) [inline promiseitwonthurt.js]: Does workshopper handle English automatically as it doesn't have the `_en` suffix?

**Comment by TimothyGu** [comment]
Overall looks good. Just a few questions here and there. I also assume you have sufficient knowledge of *la langue française*. Thanks for all the time you have put into this!

File: exercises/more_functional/problem.md (hunk 1 above)
Comment (tdd) [inline exercises/more_functional/problem.md]: I sure wouldn't miss this one. It's out of scope.

File: promiseitwonthurt.js (hunk 2 above)
Comment (tdd) [inline promiseitwonthurt.js]: Could. I just used the same `fpath` decl you'll find in most workshoppers, for consistency.

File: promiseitwonthurt.js (hunk 3 above)
Comment (tdd) [inline promiseitwonthurt.js]: Yes.

**Comment by tdd** [comment]
Well, I *am* French, and a professional translator to boot, so I guess I do alright :smile: Also, all my L10ns are vetted by another pro FR translator in a fork-local PR before I PR upstream, just for safety.

**Comment by TimothyGu** [comment]
@tdd sounds good :) Let's merge this then!

**Comment by tdd** [comment]
Awesome. Now we need @stevekane to `npm version minor` this.

By the way Steve, if you're interested in letting go of PIWH, can we devise a way for us to become able to publish this on npm, too?

File: exercises/more_functional/problem.md (hunk 1 above)
Comment (TimothyGu) [inline exercises/more_functional/problem.md]: OK removed.

**Pull request merged**